src/middlewared/middlewared/main.py

```diff
@@ -859,32 +862,41 @@ def get_schema(self, name):
             if asyncio.iscoroutinefunction(methodobj):
                 return await methodobj(*args)
             else:
-                return await self.threaded(methodobj, *args)
+                pool = None
+                if hasattr(methodobj, '_threaded'):
+                    pool = methodobj._threaded
```

Comment:
Confusing property name `_threaded` that stores thread pool

Reply:
True!